assets/src/domain/blocks/services/apollo/types.ts

```diff
@@ -0,0 +1,52 @@
+import { Entity, EntityEdge } from '@dataServices/types';
+import {
+	AttendeesList as AttendeeList,
+	EventsList as EventList,
+	DatetimesList as DatetimeList,
+	TicketsList as TicketList,
+} from '@dataServices/apollo/queries';
+
+export interface Attendee extends Entity {
+	address?: string;
+	address2?: string;
+	avatar?: string;
+	bio?: string;
+	city?: string;
+	country?: string;
+	email?: string;
+	firstName: string;
+	fullName: string;
+	lastName: string;
+	phone?: string;
+	shortBio?: string;
+	state?: string;
+	zip?: string;
+}
```

Comment:
don't have to do it now, but we should consider extracting:

- address?: string;
- address2?: string;
- city?: string;
- country?: string;
- state?: string;
- zip?: string;

into a separate `Address` type because those fields will be used for Venues as well

Reply:
I agree 👍

Reply:
Done